cypress/integration/sitewide-banner.js

```diff
@@ -23,9 +23,41 @@ describe('Site Wide Banner', () => {
   it('The Site Wide Banner is displayed on campaign pages', () => {
     cy.anonVisitCampaign(exampleCampaign);
 
+    if ('#banner-portal > .wrapper > [data-test=site-wide-banner]') {
+      cy.get('#banner-portal > .wrapper > [data-test=site-wide-banner]').should(
+        'have.length',
+        1,
+      );
+    }
```

Comment:
Let's remove the conditional here since we'd always expect this banner to appear for anonymous users on a campaign page.
```suggestion
    cy.get('#banner-portal > .wrapper > [data-test=site-wide-banner]').should(
      'have.length',
      1,
    );
```